README.md

```diff
@@ -240,13 +252,26 @@ and you should see:
 ```
 * Selenium requires **Java 8** to run
 
+#### Executing Nightwatch Tests For a Certain Environment
+A webserver is required for Nightwatch to execute. Locally, you will likely execute E2E tests while running the local development server via `npm run watch`, followed by `npm run test:e2e`.
+
+However, it is sometimes useful to execute E2E tests for a certain build-type, which is done by first running a build for that build-type, then executing the tests with the build-type passed as an argument.
+
+For example, to execute the E2E tests for Production:
+
+1. `NODE_ENV=production npm run build -- --buildtype=production`
+2. `BUILDTYPE=production npm run test:e2e`
+    - The Nightwatch startup script will see that port `3001` is not blocked (as it would be by the watch-task), and will start a [static webserver](#static-webserver) for the production build.
```

Comment:
If we are running `yarn watch`, will it still connect to that if the `BUILDTYPE=production`?

Reply:
Yeah, it always starts the static webserver at port 3001, so if that port is in use, it assumes that it must be a version of vets.gov.